app/frontend/components/pages/delegations/currentDelegationPage.js

```diff
@@ -0,0 +1,102 @@
+const {h} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../../actions')
+
+const CopyOnClick = require('../../common/copyOnClick')
+
+const CurrentDelegationPage = ({currentStakepool, undelegateStakepool}) => {
+  return h(
+    'div',
+    {class: 'current-delegation card'},
+    h('h2', {class: 'card-title'}, 'Current delegation'),
+    currentStakepool.address === ''
+      ? h(
+        'a',
+        {
+          class: 'undelegated',
+        },
+        'Your Ada is undelegated. Delegate now!'
+      )
+      : h(
+        'div',
+        {class: 'stakepool-info'},
+        h(
+          'div',
+          {class: 'stakepool-name-wrapper'},
+          h('p', {class: 'stakepool-name'}, `${currentStakepool.name}`),
+          h(
+            'button',
+            {
+              class: 'button undelegate',
+              id: 'add-pool',
+              onClick: undelegateStakepool,
+              disabled: false,
+            },
+            'Undelegate'
+          )
+        ),
+        h(
+          'div',
+          {class: 'address-value'},
+          h('p', {class: 'one-click-select'}, currentStakepool.address)
+        ),
+        h(
+          'div',
+          {class: 'address-links blockexplorer-link'},
+          h(
+            CopyOnClick,
+            {
+              value: currentStakepool.address,
+              elementClass: 'address-link copy',
+            },
+            h('a', {class: 'copy-text'}, 'Copy Address')
+          ),
+          h(
+            'div',
+            {},
+            h('span', {}, 'View on '),
+            h(
+              'a',
+              {
+                class: 'address-link',
+                href: `https://seiza.com/blockchain/address/${currentStakepool.address}`,
+                style: 'margin-right:0',
+                target: '_blank',
+                rel: 'noopener',
+              },
+              'Seiza'
+            ),
+            h('span', {}, ' | '),
+            h(
+              'a',
+              {
+                class: 'address-link',
+                href: `https://adascan.net/address/${currentStakepool.address}`,
+                style: 'margin-right:24px',
```

Comment:
whoa didn't know we had that in adalite code, it's generally a bad idea to have styles out of styles file. Please try to put this in respective css class, perhaps do it as well for the code you were inspired by.

Reply:
Came from `addressItem.js`. Fixed both.